cfgov/unprocessed/apps/youth-employment-success/js/route.js

```diff
@@ -0,0 +1,18 @@
+import { assign } from './util';
+
+/**
+ * Factory function for creating a route object. Designed to be used in conjunction
+ * with the routeOptionReducer
+ * @param {object} route optional object describing the values of the route object
```

Comment:
You can enclose the parameter name in brackets to designate to jsdocs that it's optional (see [here](https://devhints.io/jsdoc)). So should be:

```
* @param {Object} [route] optional object describing the values of the route object.
```

Reply:
oh cool, thanks!